brevent/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -34,7 +34,7 @@
     <string name="brevent_service_adb_running">（已启用）</string>
     <string name="brevent_service_usb_connected">（已连接）</string>
     <string name="brevent_service_run_as_root">使用 ROOT 启动</string>
-    <string name="brevent_service_success">黑域服务已成功启动。请点击图标，选择一些应用，然后黑域它们</string>
+    <string name="brevent_service_success">黑域服务已启动。请点按图标选择一些应用，然后使用右上角的按钮黑域它们</string>
```

Comment:
The Unicode code point for the prohibition sign (also called the no symbol) is U+20E0, Combining Enclosing Circle Backslash (  ⃠  ). It is a combining character, which means that it appears on top of the character before it, so putting d&#x20E0; will show  d⃠. (Note: This will appear only if the character exists in an available font.) It also appears in the Webdings and Wingdings 2 fonts.

所以，后面加个空格就好。